doc/cabal-commands.rst

```diff
@@ -323,10 +327,10 @@ A cabal command target can take any of the following forms:
    file. This is supported by ``build``, ``repl``, ``run``, and ``clean``.
    Script targets are not part of a package.
 
-cabal v2-build
+cabal build
 ---------------
```

Comment:
this should probably be shortened to the length of the heading? (bunch more instances of this below)